packages/database/src/DatabaseSeeder.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Tempest\Database;
+
+use UnitEnum;
+
+interface DatabaseSeeder
+{
+    public function run(null|string|UnitEnum $database): void;
```

Comment:
Need to think about this. I don't like the name "database", but it is most consistent within the rest of the framework